assets/components/subscriptionFaq/subscriptionFaq.jsx

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { subscriptionsTermsLinks } from 'helpers/legal';
+import type { SubscriptionProduct } from '../../helpers/subscriptions';
+
+
+// ----- Props ----- //
+
+type PropTypes = {|
+  subscriptionProduct: SubscriptionProduct,
+|};
+
+// ----- Functions ----- //
+
+function faqLink(props: PropTypes) {
+  if (props.subscriptionProduct === 'GuardianWeekly') {
+    return 'https://www.theguardian.com/help/2012/jan/19/guardian-weekly-faqs';
+  }
+
+  return 'https://www.theguardian.com/subscriber-direct/subscription-frequently-asked-questions';
+}
+
+// ----- Component ----- //
+
+function SubscriptionFaq(props: PropTypes) {
+
+  switch (props.subscriptionProduct) {
+    case 'GuardianWeekly':
+      return (
+        <div className="component-subscription-faq">
+          <div className="component-subscription-faq__text">
+            You may also find help in our
+            <a className="component-subscription-faq__href" href={faqLink(props)}> Frequently Asked Questions</a>.
+          </div>
+        </div>
+      );
+    case 'DigitalPack':
+      return (
+        <div className="component-subscription-faq">
+          <div className="component-subscription-faq__text">
+            You may also find help in our
+            <a className="component-subscription-faq__href" href={faqLink(props)}> Frequently Asked Questions</a> and
+            in the&nbsp;
+            <a className="component-subscription-faq__href" href={subscriptionsTermsLinks.DigitalPack}>
+            Digital Pack terms and conditions
+            </a>.
+          </div>
+        </div>
+      );
+
+    default:
+      return (
+        <div className="component-subscription-faq">
+          <div className="component-subscription-faq__text">
+            You may also find help in our
+            <a className="component-subscription-faq__href" href={faqLink(props)}> Frequently Asked Questions</a>.
+          </div>
+        </div>
+      );
+  }
+}
```

Comment:
It looks like the wrappers here are quite similar. You could use the React [`children` API](https://reactjs.org/docs/react-api.html#reactchildren):

```jsx
function Wrapper(props: { children: React$Node }) {
  return (
    <div className="component-subscription-faq">
      <div className="component-subscription-faq__text">{props.children}</div>
    </div>
  );
}
```

to make this bit re-usable, and then apply it like this:

```jsx
<Wrapper>
  You may also find help in our
  <a className="component-subscription-faq__href" href={faqLink(props)}> Frequently Asked Questions</a>.
</Wrapper>
```

This is just an option if you find it useful, I'm not sure it makes it any simpler 🤔.